api/src/test/java/io/opentelemetry/api/common/AttributesTest.java

```diff
@@ -61,7 +61,7 @@ void forEach_empty() {
     emptyAttributes.forEach(
         new AttributeConsumer() {
```

Comment:
@jkwatson Did we realize that AttributeConsumer can't be used with lambdas due to the type parameter?

Reply:
yes. This is unfortunate, but true.